client/components/common/NavBar.js

```diff
@@ -79,7 +79,7 @@ const NavBar = (props) => {
       <div className="navbar">
         <div className="navbar__bottom-section">
           <NavMenu to="/dashboard">Dashboard</NavMenu>
-          <NavMenu to="/groups">Groups</NavMenu>
+          <NavMenu to="/groups">My Groups</NavMenu>
```

Comment:
Please rebase your branch on `mvp-develop`. This item was removed on `mvp-develop`.

Reply:
Done @olusoladavid